codejail/safe_exec.py

```diff
@@ -257,6 +277,11 @@ def not_safe_exec(
     Note that `limit_overrides_context` is ignored here, because resource limits
     are not applied.
     """
+    # Because it would be bad if this function were used in production,
+    # let's log a warning when it is used.  Developers can live with
+    # one more log line.
+    log.warning("Using codejail/safe_exec.py:not_safe_exec for %s", slug)
```

Comment:
I get why this is a warning. Could the log message note that this should not be used in production?

Reply:
I made it a bit scarier, see what you think.